packages/react/src/components/CommandCenter/InputStore.ts

```diff
@@ -0,0 +1,40 @@
+import { makeAutoObservable } from "mobx";
+
+
+const DEFAULT_INPUT = {
+    placeholder: 'What would you like to do?',
+    label: 'Command',
+    value: ''
+};
+
+export class InputStore {
```

Comment:
Simple store for managing the input component. We use an external store instead of React's internal state as this allows us to use the value of the store externally.